prow/cmd/branchprotector/BUILD

```diff
@@ -4,36 +4,19 @@ load("@io_bazel_rules_k8s//k8s:object.bzl", "k8s_object")
 
 k8s_object(
     name = "oneshot",
-    cluster = "{STABLE_PROW_CLUSTER}",  # Run make :get-cluster-credentials
+    cluster = "{STABLE_PROW_CLUSTER}",
     image_chroot = "{STABLE_DOCKER_REPO}",
     images = {
-        "gcr.io/k8s-testimages/branchprotector:latest": ":branchprotector-image",
+        "gcr.io/k8s-testimages/branchprotector:latest": ":image",
```

Comment:
FWIW, I've taken to dropping the majority of the prefix and leaning on `image_chroot`.  In this case I'd probably have something like:
```python
images = {
    "branchprotected:latest": ":image",
}
```

This isn't strictly necessary if (for example) you want the configs to be consumable without Bazel, but it could run into problems with some registries (e.g. DockerHub) that don't support the multi-level names allowed in v2.

Reply:
FWIW, this is rapidly evolving best-practice, so take it with a grain of salt. :)

Reply:
Done